Dockerfile

```diff
@@ -16,10 +16,13 @@
 FROM golang:1.13 as builder
 
 WORKDIR /github.com/kubernetes-sigs/external-dns
+COPY go.mod .
```

Comment:
Please leave the the Dockerfile changes out of this PR, the PR should only focus on the Cloudflare changes like in the title.
But feel free to create a separate PR.